### ISSUE TITLE
Add `--version` flag to CLI

### ISSUE THREAD

**Issue by dhirschfeld** [microsoft/go-sqlcmd#94]
Having a  `--version` flag is pretty standard for a CLI as it lets the user know what version of the tool they're using, and hence also helps with bug reports.
```
> sqlcmd --version
sqlcmd.exe: error: unknown flag --version
```

**Comment by dhirschfeld** [comment]
Am I missing something obvious and there's another way to know what version of `sqlcmd` I'm running?

**Comment by stuartpa** [comment]
Fixed in #100
